.github/workflows/test.yaml

```diff
@@ -40,11 +40,20 @@ jobs:
 
       - uses: cachix/cachix-action@v15
         with:
-          name: holochain-wind-tunnel
-          authToken: ${{ secrets.CACHIX_HOLOCHAIN_WIND_TUNNEL }}
+          name: holochain-ci
```

Comment:
Switched to using our bigger cache. The `holochain-wind-tunnel` one was a free account and only had 5GB. That's part of the problem here.